pkg/containerinstall/install.go

```diff
@@ -0,0 +1,255 @@
+package containerinstall
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/containers/podman/v4/pkg/bindings/containers"
+	"github.com/containers/podman/v4/pkg/bindings/images"
+	"github.com/containers/podman/v4/pkg/bindings/secrets"
+	"github.com/containers/podman/v4/pkg/specgen"
+	"github.com/opencontainers/runtime-spec/specs-go"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/steps"
+)
+
+var (
+	devEnvVars = []string{
+		"AZURE_FP_CLIENT_ID",
+		"AZURE_RP_CLIENT_ID",
+		"AZURE_RP_CLIENT_SECRET",
+		"AZURE_SUBSCRIPTION_ID",
+		"AZURE_TENANT_ID",
+		"DOMAIN_NAME",
+		"KEYVAULT_PREFIX",
+		"LOCATION",
+		"PROXY_HOSTNAME",
+		"PULL_SECRET",
+		"RESOURCEGROUP",
+	}
```

Comment:
could we make this a map instead? That'd allow us to give names to the env variables instead of just appending numbers at the end of the env variables

Reply:
I'm actually appending the env variable after "ARO_", no numbers in any of it :)

Reply:
ah right, always confuse key and value stuff ><